src/allPass.js

```diff
@@ -22,4 +22,4 @@ var curry = require('./curry');
  *      f(11); //=> false
  *      f(12); //=> true
  */
-module.exports = curry(_predicateWrap(_all));
+module.exports = curry(_predicateWrap(all));
```

Comment:
It does to satisfy one of the invariants added in #836. I've switched to `_curry1`, though.